tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
Sorry, by scope wrapping I was referring to closures. Something like this:

```
var rateRollElement = $('.pid_tuning input[name="roll_rate"]');

...


function redrawRateCurves(event) {

...

    checkInput(rateRollElement);

...
```

This function is defined within other function (which is also nested), referring to a local variable created in the same scope. Because there are no public/private methods on the object, this is often abused to simulate private method behaviour. Not to mention that these functions are sometimes overriden with a different scope. This contributes to a very nested and unstructured code that is both hard to read and modify without following the same anti-patterns.